ksql-api/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -40,16 +39,14 @@
   private final Endpoints endpoints;
   private final HttpServerOptions httpServerOptions;
   private final Server server;
-  private final WorkerExecutor workerExecutor;
   private ConnectionQueryManager connectionQueryManager;
   private HttpServer httpServer;
 
   public ServerVerticle(final Endpoints endpoints, final HttpServerOptions httpServerOptions,
-      final Server server, final WorkerExecutor workerExecutor) {
+      final Server server) {
```

Comment:
nit: validate params that will be stored in object state; ensuring object does not get into an invalid state.

Reply:
ack